fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
IMHO we just want to keep the 'log' key then.

On Wed, Sep 30, 2015 at 12:33 PM, Jimmi Dyson <notifications@github.com>
wrote:

> In fluentd/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/4#discussion_r40817266>
> :
>
> > @@ -135,18 +134,36 @@
> >      bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
> >      ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
> >    </filter>
> > +  <filter **kibana**>
>
> See here
> <https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/blob/master/lib/fluent/plugin/filter_kubernetes_metadata.rb#L170-L178>
> - checks first & last chars, parses.
>
> It also removes the log field here
> <https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/blob/master/lib/fluent/plugin/filter_kubernetes_metadata.rb#L175>
> - could remove that if you'd like.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/4/files#r40817266>
> .
>